junit-jupiter-api/src/main/java/org/junit/jupiter/api/RepetitionInfo.java

```diff
@@ -50,4 +51,24 @@ public interface RepetitionInfo {
 	 */
 	int getTotalRepetitions();
 
+	/**
+	 * Get the current number of repetitions of the corresponding
+	 * {@link RepeatedTest @RepeatedTest} method that have ended in a failure.
+	 *
+	 * @since 5.10
+	 * @see RepeatedTest#failureThreshold()
```

Comment:
❔ Should this link to `getFailureThreshold()` below instead?

Reply:
It could. I don't think it makes much of a difference, but I can change that.